README.md

```diff
@@ -1,2 +1,126 @@
+| InfoLevel | SuperMetrics | 3
+| InfoLevel | ServiceStatus | 3
+| InfoLevel | CustomGroups | 3
+
+There are 6 levels (0-5) of detail granularity for each section as follows;
+
+| Setting | InfoLevel | Description |
+| ------- | ---- | ----------- |
+| 0 | Disabled | does not collect or display any information
+| 1 | Summary** | provides summarised information for a collection of objects
+| 2 | Informative | provides condensed, detailed information for a collection of objects
+| 3 | Detailed | provides detailed information for individual objects
+| 4 | Adv Detailed | provides detailed information for individual objects, as well as information for associated objects
+| 5 | Comprehensive | provides comprehensive information for individual objects, such as advanced configuration settings
+
+\*\* *future release*
+
+### Healthcheck
+The **Healthcheck** sub-schema is used to toggle health checks on or off. Currently none configured.
+
+## Examples 
+- Generate HTML & Word reports with Timestamp
+Generate a vROps As Built Report for vROps Server 'vrops-01.corp.local' using specified credentials. Export report to HTML & DOC formats. Use default report style. Append timestamp to report filename. Save reports to 'C:\Users\Tim\Documents'
+```powershell
+New-AsBuiltReport -Target 'vrops-01.corp.local' -Username 'admin' -Password 'VMware1!' -Report VMware.vROps -Format Html,Word -OutputPath 'C:\Users\Tim\Documents' -Timestamp
+```
+- Generate HTML & Text reports with Health Checks
+Generate a vROps As Built Report for vROps Server 'vrops-01.corp.local' using stored credentials. Export report to HTML & Text formats. Use default report style. Highlight environment issues within the report. Save reports to 'C:\Users\Tim\Documents'
+```powershell
+New-AsBuiltReport -Target 'vrops-01.corp.local' -Credentials $Creds -Report VMware.vROps -Format Html,Text -OutputPath 'C:\Users\Tim\Documents' -EnableHealthCheck
+```
+- Generate report with multiple vROps Servers using Custom Style
+Generate a single vROps As Built Report for vROps Servers 'vrops-01.corp.local' and 'vrops-02.corp.local' using specified credentials. Report exports to WORD format by default. Apply custom style to the report. Reports are saved to the user profile folder by default.
+```powershell
+New-AsBuiltReport -Target 'vrops-01.corp.local','vcenter-02.corp.local' -Username 'admin' -Password 'VMware1!' -Report VMware.vROps -StylePath C:\Scripts\Styles\MyCustomStyle.ps1
+```
+- Generate HTML & Word reports, attach and send reports via e-mail
+Generate a vROps As Built Report for vROps Server 'vrops-01.corp.local' using specified credentials. Export report to HTML & DOC formats. Use default report style. Reports are saved to the user profile folder by default. Attach and send reports via e-mail.
+```powershell
+New-AsBuiltReport -Target 'vrops-01.corp.local' -Username 'admin' -Password 'VMware1!' -Report VMware.vROps -Format Html,Word -OutputPath C:\Users\Tim\Documents -SendEmail
```

Comment:
Move `Report` parameter position to be first

Reply:
Fixed